### DISCUSSION TITLE
Customer support

### DISCUSSION THREAD

**Discussion by robdesautel** [Kong/insomnia#8048] [Feedback]
I have been trying to contact someone for customer support. There has been no success. I have emailed support@insomnia.rest, reached out on slack, left a comment on a previous issue that's closed from last year, spoken to someone in the enterprise sales team, and when sent an email to suppor@konghq.com all with zero success.

This is a horrid customer service model. For the billionth time as a last resort before I seek separate legal advice, who do I contact about billing because 100% nothing works.......

**Comment by subnetmarco** [comment]
@robdesautel let me escalate this on our end, I see your support request opened 4 days ago. It may take up to a week to get back to you, especially if there is a weekend in-between. You will hear back in the support ticket.
